targets/nano-33-ble-s140v7.json

```diff
@@ -0,0 +1,8 @@
+{
+	"inherits": ["nrf52840", "nrf52840-s140v7"],
+	"build-tags": ["nano_33_ble-s140v7", "nano_33_ble"],
```

Comment:
What I did for other boards, was that I inherited two files: the board target (in this case `nano-33-ble`) and the chip+SoftDevice target (in this case `nrf52840-s140v7`). That means that most board-specific configurations can be kept per board.

Something like this (untested):

```json
{
  "inherits": ["nano-33-ble", "nrf52840-s104v7"],
  "flash-command": "nrfjprog -f nrf52 --sectorerase --program {hex} --reset",
  "openocd-interface": "jlink"
}
```

Reply:
Thanks, will fix